test/integration/connect/envoy/helpers.bash

```diff
@@ -152,6 +152,37 @@ function assert_envoy_version {
   echo $VERSION | grep "/$ENVOY_VERSION/"
 }
 
+function assert_envoy_expose_checks_listener_count {
+  local HOSTPORT=$1
+  local EXPECT_PATH=$2
+
+  # scrape this once
+  BODY=$(get_envoy_expose_checks_listener_once $HOSTPORT)
+  echo "BODY = $BODY"
+
+  CHAINS=$(echo "$BODY" | jq '.active_state.listener.filter_chains | length')
+  echo "CHAINS = $CHAINS (expect 1)"
+  [ "${CHAINS:-0}" -eq 1 ]
+
+  RANGES=$(echo "$BODY" | jq '.active_state.listener.filter_chains[0].filter_chain_match.source_prefix_ranges | length')
+  echo "RANGES = $RANGES (expect 3)"
+  [ "${RANGES:-0}" -eq 3 ]
```

Comment:
Is this expected to be 2 on a IPv4 only system with `conf.all.disable_ipv6=1` ?

Reply:
Yes. I believe our golden tests for `agent/xds` will fail on an ipv6-less machine so at least these two sets of tests would break at the same time?

Reply:
Might be worth a comment at the head of the test explaining that for the future reader.

Reply:
:+1: